addons/info.go

```diff
@@ -0,0 +1,50 @@
+package addons
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/db"
+)
+
+func Info(app, addon string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+	addonInfo, err := c.AddonShow(app, addon)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
+	}
+
+	dbInfo, err := db.Show(app,addon)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
+	}
+
+	forceSsl, internetAccess := "disabled", "disabled"
+	if featuresLen := len(dbInfo.Features); featuresLen > 0 {
```

Comment:
nitpick: we don't really use the construct you used as we don't find it easily readable.
```suggestion
	featuresLen := len(dbInfo.Features)
	if featuresLen > 0 {
```